sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_day_6_v1/query.sql

```diff
@@ -1,37 +1,31 @@
 WITH pop AS (
   SELECT
-    ROW_NUMBER() OVER (PARTITION BY client_id) AS rn,
     client_id,
-    normalized_country_code AS country_code,
+    country AS country_code,
     normalized_channel AS channel,
-    application.build_id AS build_id,
+    app_build_id AS build_id,
     normalized_os AS os,
     mozfun.norm.truncate_version(normalized_os_version, "minor") AS os_version,
-    environment.settings.attribution.source AS attribution_source,
-    environment.partner.distribution_id AS distribution_id,
-    COALESCE(environment.settings.attribution.ua, '') AS attribution_ua,
-    DATE(submission_timestamp) AS date
-  FROM
-    telemetry.new_profile
-  WHERE
-    DATE(submission_timestamp) = DATE_SUB(@submission_date, INTERVAL 6 day)
-    AND payload.processes.parent.scalars.startup_profile_selection_reason = 'firstrun-created-default'
-),
-dist_pop AS (
-  -- make sure that we only get one entry per client
-  SELECT
-    * EXCEPT (rn)
+    attribution_source,
+    distribution_id,
+    attribution_ua,
+    first_seen_date AS date
   FROM
-    pop
+    telemetry.clients_first_seen_v2
   WHERE
-    rn = 1
+    first_seen_date = DATE_SUB(@submission_date, INTERVAL 27 day)
```

Comment:
We will now have a reporting delay for the last funnel steps of 28 days since we need to observe for that long to get the complete returned second day and returned 4th week outcomes for each cohort